idaes/ui/flowsheet.py

```diff
@@ -292,6 +294,7 @@ def _add_unit_model_with_ports(self, unit):
                 performance_df["Value"] = performance_df["Value"].map(
```

Comment:
Minor remark (unrelated to the changes in question): the `lambda` here looks redundant, and one could use `value()` directly:

```py
performance_df["Value"] = performance_df["Value"].map(value)
```

Reply:
Removed.

Reply:
Okay, the lambda is necessary because I'm applying a function to all of the values in the pandas Series. When I remove it the tests fail.

Reply:
Sorry for the confusion. What I meant with "removing the lambda" was to use directly the `value` function (i.e. the function object itself), rather than calling it, as an argument to `Series.map()` (which is how it was used in the commit where the lambda was removed, from what I can see from the errors log). In other words, `my_func` and `my_other_func` below are equivalent, which is what I was referring to with "the lambda looks redundant":

```py
my_func = lambda x: value(x)
my_other_func = value
```

Reply:
Oh okay, that makes more sense. I'm sorry I didn't understand. I removed the lambda and just used `value`